testdata/p4_16_samples_outputs/invalid-union-frontend.p4

```diff
@@ -0,0 +1,39 @@
+header E {
+}
+
+header H {
+    bit<32> f;
+}
+
+header_union HU {
+    E e;
+    H h;
+}
+
+parser p(inout HU hu) {
+    state start {
+        hu = (HU){#};
+        transition accept;
+    }
+}
+
+control c(inout HU hu) {
+    @name("c.h") H h_0;
+    @name("c.a") action a() {
+    }
+    apply {
+        a();
+        if (hu.isValid()) {
```

Comment:
I do not see in the spec anywhere that `hu.isValid()` is defined for values `hu` of type header_union.  Do you?

Reply:
Ugh.  I found it.  It is defined.  Resolving this issue.